test/test-exception.js

```diff
@@ -5,24 +5,33 @@ if (process.argv[2] === 'child') {
   require('../');
 
   function myException(request, response) {
-    const m = '*** exception.js: testcase exception thrown from myException()';
-    throw new UserException(m);
-  }
-
-  function UserException(message) {
-    this.message = message;
-    this.name = 'UserException';
+    const m = '*** test-exception.js: throwing uncaught Error';
+    throw new Error(m);
   }
 
   myException();
+
 } else {
   const common = require('./common.js');
   const spawn = require('child_process').spawn;
   const tap = require('tap');
 
   const child = spawn(process.execPath, [__filename, 'child']);
+  // Capture stderr output from the child process
+  var stderr = '';
+  child.stderr.on('data', (chunk) => {
+    stderr += chunk;
+  });
   child.on('exit', (code) => {
-    tap.plan(2);
+    tap.plan(4);
+    const v8_version = (process.versions.v8).match(/\d+/g);
+    if (v8_version[0] <= 5 && v8_version[1] < 4) {
```

Comment:
Same as for the equivalent check in the C++ code -- v8 v4.5 would fail this check.